libs/designsystem/slide/src/slides.component.ts

```diff
@@ -93,6 +96,13 @@ export class SlidesComponent implements OnInit, AfterViewInit {
     this.swiperContainer.nativeElement.swiper.slideTo(index);
   }
 
+  ngOnChanges(changes: SimpleChanges): void {
+    if (changes?.slides.firstChange === false) {
```

Comment:
Move conditional prop accessor:
```suggestion
    if (changes.slides?.firstChange === false) {
```